README.md

```diff
@@ -0,0 +1,29 @@
+# zabbix-alertmanager
+
+Project consists of 2 parts:
+## 1. zabbix-sender
+Sender catches the alerts from Prometheus alertmanager and pushes them to the Zabbix server by using trapper items.
+
+## 2. zabbix-provisioner
+The provisioner will load the current configured rules from a directory and will create hostgroup/host/application/items/triggers accordingly, 
```

Comment:
`configured rules` -> `Prometheus Alerting rules` and link to the description https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/